GVFS/GVFS.Service/Program.Mac.cs

```diff
@@ -1,7 +1,10 @@
 ﻿using GVFS.Common;
+using GVFS.Common.FileSystem;
 using GVFS.Common.Tracing;
 using GVFS.PlatformLoader;
 using System;
+using System.IO;
+using System.Linq;

```

Comment:
These three new `using` statements are not needed:

using GVFS.Common.FileSystem;
using System.IO;
using System.Linq;

Reply:
These are required in `GVFS/GVFS.Service/Program.Mac.cs`. Keeping.
These are Not required in `GVFS/GVFS.Service/GVFSService.Mac.cs`. Removed it.

Reply:
Ahh sorry about that, yep I left my original comment in the wrong file.